datasets/ua/war_sanctions/crawler.py

```diff
@@ -34,6 +39,19 @@ def extract_label_value_pair(label_elem, value_elem, data):
     return label, value
 
 
+def apply_dob_pob(entity, dob_pob):
+    if dob_pob:
```

Comment:
Just an aesthetic point: if you do `if dob_pob` then the whole rest of the function is one indented. If, instead, you do `if not dob_pob` you can just put a `return` and keep the function more legible.